examples/src/pages/Migration.tsx

```diff
@@ -360,6 +360,8 @@ export function Migration() {
 
       const inspectNodeUi = new InspectNodeUI(gui.addFolder('Last clicked node'), client, viewer);
 
+      viewer.renderer.setPixelRatio(window.devicePixelRatio);
```

Comment:
you are setting pixel ratio at https://github.com/cognitedata/reveal/blob/2644eb4d78284cda71d6fcab8e56abf28c692cff/viewer/packages/rendering/app/index.ts#L61 why do we need to set them here